clients/privacy-center/app/server-environment.ts

```diff
@@ -115,9 +172,14 @@ export const loadConfigFromFile = async (
   ];
   const file = await loadConfigFile(urls);
   if (file) {
-    const config = JSON.parse(file) as Config;
-    // DEFER: validate the configuration here, log helpful warnings, etc.
+    const config = transformConfig(JSON.parse(file));
+    const { isValid, message } = validateConfig(config);
+    // DEFER: add more validations here, log helpful warnings, etc.
     // (see https://github.com/ethyca/fides/issues/3171)
+    if (!isValid) {
+      console.warn("Configuration file is invalid! Message:", message);
```

Comment:
maybe we aren't planning to do this until https://github.com/ethyca/fides/issues/3171, but it's a little confusing to have the error log out to the console but only something generic in the UI. for example, if I make my config invalid by having two `executable: true`s, then I get, in my terminal window, the expected `Configuration file is invalid! Message: Cannot have more than one consent option be executable` (which, as an aside, looks just like a log statement in my terminal for some reason, not a different color or anything... maybe that's a turbo config?) 

![image](https://user-images.githubusercontent.com/24641006/236847646-7c1c7661-4d27-4583-9ad6-45bc7cefd58c.png)


but in the UI all I get is 

![image](https://user-images.githubusercontent.com/24641006/236847461-7aca7bfa-0469-474c-943d-9d8f04db201f.png)

which doesn't really give any clues on what is going wrong

Reply:
So I *think* this is the right balance; the "detailed" error message is shown on the server logs, and the client-side doesn't load and gives a generic (but accurate) error message.

If you imagine this happening in a scenario where you have 1,000s of visitors to an active privacy center online and it gets misconfigured, you'd likely only want consumers to see a generic message.